packages/mask/src/plugins/Profile/SNSAdaptor/NFTPage.tsx

```diff
@@ -50,6 +58,27 @@ export function NFTPage(props: NFTPageProps) {
     const { type, name, address } = value ?? {}
     const { loading: loadingWalletGun, value: walletAddressGun } = useUserOwnerAddress(identity.identifier.userId)
 
+    const rulesTipMap = [
+        t('plugin_profile_binding_rule1'),
+        t('plugin_profile_binding_rule2'),
+        t('plugin_profile_binding_rule3'),
+        t('plugin_profile_binding_rule4', { suffix: `".eth"` }),
+    ]
+
+    const tooltipRender = (
```

Comment:
Sorry forgotten to say that MUI has the Tooltip component.

Learn more: https://github.com/DimensionDev/Maskbook/pull/4814